packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-draggable-dummy-answer.tsx

```diff
@@ -0,0 +1,53 @@
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
+import { faCircleXmark } from '@fortawesome/free-solid-svg-icons'
+import {
+  KeyboardEventHandler,
+  type ChangeEventHandler,
+  type MouseEventHandler,
+} from 'react'
+
+import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { cn } from '@/helper/cn'
+
+interface BlankDraggableDummyAnswerProps {
+  text: string
+  onChange: ChangeEventHandler<HTMLInputElement>
+  onKeyDown: KeyboardEventHandler
+  onRemoveClick: MouseEventHandler
+}
+
+export function BlankDraggableDummyAnswer(
+  props: BlankDraggableDummyAnswerProps
+) {
+  const { text, onChange, onKeyDown, onRemoveClick } = props
+  const blanksExerciseStrings = useEditorStrings().plugins.blanksExercise
+
+  return (
+    <div className="relative">
+      <span className="serlo-autogrow-input mb-1 mr-2" data-value={text + '_ '}>
```

Comment:
Why is the `+ '_' ` needed?

Reply:
Because without it the autogrow input becomes too small for its content

Reply:
Ah, I get it! Is this the case everywhere we use the `serlo-autogrow-input`? Or only here?

Reply:
Everywhere, I think

Reply:
Everywhere. Only we use `'_ '` here to have space for the delete button 🙈

Reply:
_(Also `'_'` is a cool guy, obviously.)_

Reply:
All right! Thanks! :)